docs/mocksupport.rst

```diff
@@ -878,3 +878,64 @@ DMTF CIM schema download support
       :attributes:
 
    .. rubric:: Details
+
+
+.. _`Mock inmemory CIM repository`:
+
+Mock inmemory CIM repository
+----------------------------
+
+pywbem_mock provides a standard interface to a CIM repository that
+stores CIM classes, CIM instances, and CIM qualifier declarations.
+
+This repository implements a repository in memory that is created and
+destroyed each time the FakedWBEMConnection class is constructed and
+destroyed.
+
+The following classes implement this repository.
+
+.. _`Inmemory CIM Repository class`:
+
+Inmemory CIM Repository class
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. automodule:: pywbem_mock._inmemoryrepository
```

Comment:
I suggest to remove this `automodule` directive. It does not provide much value and also it applies to the whole module while this section is scoped to just one class of the module.

Reply:
CLOSED - Comment became moot since the module docstring is now used just once.

Reply:
CLOSED - became moot since the last commit changed the module docstring.